fr/releases/centreon-commercial-extensions.md

```diff
@@ -18,6 +18,17 @@ commerciales, veuillez contacter le support.
 
 ## Centreon MAP release notes
 
+### 20.10.2
+
+* [Amelioration] Le protocole TLS pour communiquer avec le composant Broker a été implémenter
+* [Fix] Mettre à jour depuis des versions précédentes causait parfois un échec lors du redémarrage du service centreon-map
+* [Fix] Une mise à jour ou une nouvelle installation causait parfois une mauvaise attribution des droits sur le dossier media
+* [Fix] Ajouter ou supprimer un Downtown depuis l'interface web causait parfois une rupture de connexion avec Broker
```

Comment:
[Fix] Ajouter ou supprimer un Downtime depuis l'interface web causait parfois une rupture de connexion avec Broker

Reply:
I see no difference between the two